appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+
+platform:
+    - x86
+
+environment:
+    PUSH_IMAGE: 'rustops/crates-build-env-windows'
+
+# If Docker is currently configured to run Linux containers, we need to switch
+# to running Windows containers.
+#
+# https://github.com/docker/cli/issues/1042
+init:
+    # - ps: $env:ProgramFiles/Docker/Docker/DockerCli.exe -SwitchDaemon
+
+build_script:
+    - ps: ./windows/ci/build.ps1
+
+deploy_script:
```

Comment:
While this is not executed on PRs, I think if I push a `foo` branch it will be deployed. Can you restrict this only to the master branch?

Reply:
Ah yes, an oversight on my part. I pushed new changes.